galsim/chromatic.py

```diff
@@ -164,10 +384,24 @@ def drawImage(self, bandpass, image=None, integrator=None, **kwargs):
                                 for details.)  [default: None]
         @param integrator       One of the image integrators from galsim.integ [default: None,
                                 which will try to select an appropriate integrator automatically.]
+                                This keyword is ignored in the case that optimized interpolation
+                                between precomputed images is to be used to render the image.
```

Comment:
Simpler: This keyword is ignored if `setupInterpolation` has been called for this object.